src/main/java/ui/diet/dietmanager/DietManagerUi.java

```diff
@@ -20,8 +20,9 @@ public static void printBye() {
 
     public String getInput() {
         Scanner in = new Scanner(System.in);
-        assert !in.nextLine().isEmpty();
-        return in.nextLine();
+        String check = in.nextLine();
```

Comment:
Can try use/adapt the getInput method from Khoa's Ui package. That may standardise it